imports/ui/components/featured-group/featured-group.html

```diff
@@ -0,0 +1,34 @@
+<template name="featuredGroup">
+	{{#if showFeaturedGroup}}
+		<div class="container">
+			<div class="featured-group">
+				<div class="featured-group-header">
+					<i class="fa fa-fw fa-star" aria-hidden="true"></i>
+					{{#mf KEY='featuredGroup.heading' NAME=regionName}}
+						Featured group in {NAME}
```

Comment:
Consider using `{REGION}` instead of `{NAME}` so translators know what's going on.